pkg/upgraders/upgradeable.go

```diff
@@ -41,9 +63,33 @@ func (c *clusterUpgrader) IsUpgradeable(ctx context.Context, logger logr.Logger)
 	}
 
 	// if the upgradeable is false then we need to check the current version with upgrade version for y-stream update
+	isRosaCluster, err := c.isRosaCluster(ctx)
+	if err != nil {
+		return false, err
+	}
+
+	var kbArticleNumber int
+
+	if isRosaCluster {
+		kbArticleNumber = 6541901
+	} else {
+		kbArticleNumber = 6657541
+	}
```

Comment:
Can these numbers please be turned into const declarations outside of the function?

Reply:
Sure, will turn them into `const`